tests/test_api.py

```diff
@@ -73,7 +72,13 @@ def iso2timestamp(iso_date: str) -> float:
 
 def assert_same_datetime(date1: str, date2: str, max_diff_seconds=5) -> None:
     """Assert that the two given dates in iso format are very close."""
-    assert abs(iso2timestamp(date2) - iso2timestamp(date1)) < max_diff_seconds
+    assert (
+        abs(
+            iso2timestamp(date2.replace("Z", "+00:00"))
```

Comment:
I guess this is some peculiar python issue?

Reply:
It seems that since pydantic v2 datetimes are output as isoformat with the Z shortcut (equivalent to +0) when outputting json. However, the standard isoformat conversion method does output +0.